test/releases/0.83.0.go

```diff
@@ -0,0 +1,105 @@
+package releases
+
+import (
+	cnao "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/shared"
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/components"
+)
+
+const secondaryDNSDeployment = "secondary-dns"
```

Comment:
Note that here we will still have this const
on future versions we won't
It isn't that important basically imo